service-loadbalancer/service_loadbalancer.go

```diff
@@ -430,6 +430,12 @@ func main() {
 
 	var kubeClient *client.Client
 	var err error
+
+	_, err = newSyslogServer("/var/run/haproxy.log.socket")
```

Comment:
Oh I meant basically add a -v or -syslog flag to (https://github.com/kubernetes/contrib/blob/master/service-loadbalancer/service_loadbalancer.go#L108) and only start the syslog server if it's defined. I don't mind a follow up pr for this if you'd rather get this one in first.